pkg/utils/experimentHelpers.go

```diff
@@ -84,11 +112,11 @@ func (expDetails *ExperimentDetails) ValidateSecrets(clients ClientSets) error {
 			log.Infof("Incomplete Information in Secret, will abort execution")
```

Comment:
this isn't really aborting, rather, moving onto the next experiment right? lets reword

Reply:
Done.